### PR TITLE
added note about GPO location change

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#5418

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md [hunk 1]
```diff
@@ -36,7 +36,9 @@ You can create a Group Policy or mobile device management (MDM) policy that will
 
 The following table lists the Group Policy settings that you can configure for Windows Hello use in your workplace. These policy settings are available in both **User configuration** and **Computer Configuration** under **Policies** &gt; **Administrative Templates** &gt; **Windows Components** &gt; **Windows Hello for Business**.
 
-
+>[!NOTE]
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md]: (MD compatibility space wanted)
```suggestion
> [!NOTE]
```

File: windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md [hunk 2]
```diff
@@ -36,7 +36,9 @@ You can create a Group Policy or mobile device management (MDM) policy that will
...
+> Starting with Windows 10 1709, location of PIN complexity section of Group Policy is **Computer Configuration** &gt; **Administrative Templates** &gt; **System** &gt; **PIN Complexity**.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md]:
```suggestion
> Starting with Windows 10 version 1709, the location of the PIN complexity section of Group Policy is: **Computer Configuration** &gt; **Administrative Templates** &gt; **System** &gt; **PIN Complexity**.
```

**Comment by illfated** [comment]
#sign-off

File: windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md [hunk 3]
```diff
@@ -36,7 +36,9 @@ You can create a Group Policy or mobile device management (MDM) policy that will
 
 The following table lists the Group Policy settings that you can configure for Windows Hello use in your workplace. These policy settings are available in both **User configuration** and **Computer Configuration** under **Policies** &gt; **Administrative Templates** &gt; **Windows Components** &gt; **Windows Hello for Business**.
 
-
+> [!NOTE]
+> Starting with Windows 10 version 1709, the location of the PIN complexity section of Group Policy is: **Computer Configuration** &gt; **Administrative Templates** &gt; **System** &gt; **PIN Complexity**.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md]:
```suggestion
> Starting with Windows 10, version 1709, the location of the PIN complexity section of the Group Policy is: **Computer Configuration** &gt; **Administrative Templates** &gt; **System** &gt; **PIN Complexity**.
```

**Comment by mypil** [comment]
@mapalko @Dansimp - PR has been copy edited and is ready for final review. This is based on issue #5418

Thank you.

cc: @kenwith

**Comment by illfated** [comment]
The PR has been approved by the document author.
Final review & merge can be done by the doc manager.

**Pull request merged**